styles/tutorsStyle.css

```diff
@@ -26,3 +29,6 @@ th {
     font-size: .7em;
 }
 
+.btn-file input[type=file] {
```

Comment:
I don't know how alphabetizing style rules works with 2 selectors. Maybe these should go in order but im not sure

Reply:
This has been re-ordered